content/en/contribute/medic/product-development-process/deploy-on-eks.md

```diff
@@ -85,23 +85,37 @@ After you have created a ticket per "Request permission" above, you should get a
    ```
 
    If get an error `no context exists with the name`, change `use-context` to `set-context` in the  command.  This will create the entry the first time.  Subsequent calls should use `use-context`.
-3. Create a new `values.yaml` file by [copying this one](https://github.com/medic/medic-infrastructure/blob/master/terraform/aws/dev/cht-projects/alpha-dev-cht-deploy-values.yaml). Be sure to update these values after you create it: 
-   * `alpha-dev` values to `USERNAME-dev` 
+3. Create a new `values.yaml` file by [copying this one](https://github.com/medic/medic-infrastructure/blob/master/terraform/aws/dev/cht-projects/alpha-dev-cht-deploy-values.yaml). Be sure to update these values after you create it:
+
+   {{< tabpane text=true >}}
+   {{% tab header="Single node CouchDB" %}}
+   * `alpha-dev` values to `USERNAME-dev`
    * Update `certificate` to the latest value from SRE - currently it's `arn:aws:iam::720541322708:server-certificate/2024-wildcard-dev-medicmobile-org-chain`
-   * Add a strong `password` - this instance is exposed to the Internet!
+   * Add a strong `password` - this instance is exposed to the Internet! \*
    * Put a UUID in `secret` - the command `uuidgen` is great for this
    * Update `host` to be your `username`.  For example: `mrjones.dev.medicmobile.org`
-4. Use `uuidgen` to fill in the `secret` in `values.yaml`
-5. Use a good passphrase (diceware!) to fill in `password` in `values.yaml`. _Please note that a few special characters are unsupported in this field like `:`, `@`, `"`, `'`, etc. Add your password as a string by enclosing it in quotes `""`, and do not use spaces in your password. This will not impact the deployment but will not let you log in to the CHT instance._
-6. Ensure you have the latest code of `cht-core` [repo](https://github.com/medic/cht-core):
+   {{% /tab %}}
+   {{% tab header="Multi node CouchDB" %}}
+   * `alpha-dev` values to `USERNAME-dev`
+   * Update `certificate` to the latest value from SRE - currently it's `arn:aws:iam::720541322708:server-certificate/2024-wildcard-dev-medicmobile-org-chain`
```

Comment:
Good idea to note that this cert is only for the dev cluster. To avoid the likelihood of someone using this for prod.

Reply:
I think given we only say the EKS docs are for dev and step 2 is very explicit:

> 2. Ensure you’re using dev EKS cluster: `kubectl config use-context arn:aws:eks:eu-west-2:720541322708:cluster/dev-cht-eks`

I don't think this extra comment is needed.  Further, I'm hoping to [skip this step](https://github.com/medic/medic-infrastructure/pull/1078) all together soon! (feel free to approve that PR if you wanna make this step better ;)